pipeline/5/Jenkinsfile-deployment-tier-1.groovy

```diff
@@ -0,0 +1,154 @@
+                    "sutConf=conf/${cephVersion}/cephadm/tier1_3node_cephadm_bootstrap.yaml",
+                    "testSuite=suites/${cephVersion}/cephadm/tier1_skip_dashboard.yaml",
+                    "addnArgs=--post-results --log-level debug"
+                ]) {
+                    rc = sharedLib.runTestSuite()
+                    test_results["Bootstrap_skip_dashboard_and_custom_ceph_config_organisation"] = rc
+                }
+            }
+        }
+    },
+    'Cephadm_Apply_Spec': {
+        stage('Cephadm suite - Apply Service spec File') {
+            sleep(360)
+            script {
+                withEnv([
+                    "sutVMConf=conf/inventory/rhel-8.3-server-x86_64.yaml",
+                    "sutConf=conf/${cephVersion}/cephadm/tier1_3node_cephadm_bootstrap.yaml",
+                    "testSuite=suites/${cephVersion}/cephadm/tier1_service_apply_spec.yaml",
+                    "addnArgs=--post-results --log-level debug"
+                ]) {
+                    rc = sharedLib.runTestSuite()
+                    test_results["Cephadm_Apply_Spec"] = rc
+                }
+            }
+        }
+    },
+    'Cephadm_Upgrade_Build_to_Build_5x': {
+        stage('Upgrade suite - Build to Build(B2B)') {
+            sleep(360)
+            script {
+                withEnv([
+                    "sutVMConf=conf/inventory/rhel-8.3-server-x86_64.yaml",
+                    "sutConf=conf/${cephVersion}/cephadm/tier1_3node_cephadm_bootstrap.yaml",
+                    "testSuite=suites/${cephVersion}/cephadm/tier1_cephadm_upgrade.yaml",
+                    "addnArgs=--post-results --log-level debug"
+                ]) {
+                    rc = sharedLib.runTestSuite()
+                    test_results["Cephadm_Upgrade_Build_to_Build_5x"] = rc
+                }
+            }
+        }
+    }
+]
+
+// Pipeline script entry point
+node(nodeName) {
+
+    timeout(unit: "MINUTES", time: 30) {
+        stage('Install prereq') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: '*/master']],
+                doGenerateSubmoduleConfigurations: false,
+                extensions: [[
+                    $class: 'SubmoduleOption',
+                    disableSubmodules: false,
+                    parentCredentials: false,
+                    recursiveSubmodules: true,
+                    reference: '',
+                    trackingSubmodules: false
+                ]],
+                submoduleCfg: [],
+                userRemoteConfigs: [[url: 'https://github.com/red-hat-storage/cephci.git']]
+            ])
+            script {
+                sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+                sharedLib.prepareNode()
+            }
+        }
+    }
+
+    timeout(unit: "MINUTES", time: 180) {
+        parallel testStages0
+    }
+
+    timeout(unit: "MINUTES", time: 120) {
+        parallel testStages1
+    }
+
```

Comment:
Please include a stage to send email. That is
```
    stage('Publish Results') {
        script {
            sharedLib.sendEMail("Deploy-Tier-1", test_results)
        }
    }
```

Reply:
Hi @psathyan,
As discussed in last meeting, we planned not to include publish section. So RGW jenkins config went in without email section. https://github.com/red-hat-storage/cephci/blob/master/pipeline/5/jenkinsfile-object-tier-1.groovy

Let me know if we have to include this again.

Reply:
Updated.